docs/guides/pushing-images-to-ECR.md

```diff
@@ -0,0 +1,92 @@
+# Pushing images to ECR
```

Comment:
We should link this page from `guides/auth.md` - it'll help people find this.

Also, shall we put this in the TOC? Not sure how to best categorize guides that are not related to the core functionality (but nevertheless very important).

Reply:
Yep was planning on it once I finished the other two. Just wanted to get the raw content up for eyes on it first.